canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.scss

```diff
@@ -52,15 +52,46 @@ $primary-tab-height: $spacing-08;
 	}
 }
 
-.properties-category-container {
-	position: relative;
-	&:last-of-type {
-		.properties-category-content.show {
-			border-bottom: 0;
+.properties-categories {  
+	.bx--accordion__item {
+		margin: 3 -1rem; // Remove 16px outer padding for accordion title and content
+	}
+	.bx--accordion__item--active .bx--accordion__content {
+		// 24px padding between accordion title and content (issue #3106)
+		padding: $spacing-06 $spacing-05 $spacing-05;
+		
+		
+		.bx--accordion__item {
```

Comment:
Can you give some details as to why we need to set each of these?

Reply:
This spacing is to keep it consistent with Accordions included in twisty panel and also with previous layout
https://github.com/elyra-ai/canvas/blob/59dc2d51940492e89d774555354243eb050ba5fe/canvas_modules/common-canvas/src/common-properties/panels/twisty/twisty.scss#L27

Reply:
This is fine but does this need to use the carbon class?  Also, can you add a comment to why each of these carbon overrides are needed?  For example why do we need to set `transform: rotate(90deg);`?

Reply:
Since Accordion inside Accordions are not supported in Carbon Components https://github.com/carbon-design-system/carbon/issues/4768#issuecomment-699066716 I am handling collapsed and expanded state to hide/unhide the inner accordion content and rotate accordion arrow present beside the header using these carbon classes.